acceptance/tests/facts/windows.rb

```diff
@@ -15,12 +15,24 @@
   if agent['platform'] =~ /2003/
     os_version  = '2003 R2'
     kernel_version = '5.2'
-  elsif agent['platform'] =~ /2008/
+  elsif agent['platform'] =~ /2008r2/
     os_version  = '2008 R2'
     kernel_version = '6.1'
-  elsif agent['platform'] =~ /2012/
+  elsif agent['platform'] =~ /2008/
+    os_version  = '2008'
+    kernel_version = '6.0'
+  elsif agent['platform'] =~ /2012r2/
```

Comment:
Should the regex's be more specific?

e.g.  `2012r2` will match `/2012/
 perhaps use `/-2012$/` ?

Also `/8.1/`  the `.` should be escaped